src/prop/minisat/core/Solver.cc

```diff
@@ -223,12 +225,16 @@ Solver::Solver(CVC4::prop::TheoryProxy* proxy,
       propagation_budget(-1),
       asynch_interrupt(false)
 {
+  d_unsatCoresProofs =
```

Comment:
I think this option is fine, but when we delete the old proof code we would delete it, since in the unsat cores + proofs mode what is necessary is what is already guarded by `isProofsEnabled`